tox.ini

```diff
@@ -36,19 +42,6 @@ conda_deps=
 commands=
     sphinx-build docs docs/_build
 
-[testenv:coverage]
```

Comment:
This deletes the `coverage` test environment? Just to check my understanding, after this the coverage tests are ran in the environments that it automatically configures based on the names above? So it doesn't need to have a specific environment created specifically. And uses the commands that are defined on the lines starting with "cov:". Yes?

Reply:
Yes exactly. I moved the coverage info up to the main `[testenv]` section and everything starting with `cov:` is what it'll call for coverage runs.